src/components/OneTrustScript.tsx

```diff
@@ -0,0 +1,12 @@
+import Script from 'next/script'
+
+export default function OneTrustScript() {
+    return (
+        <>
+            <Script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" strategy='beforeInteractive' type="text/javascript" charSet="UTF-8" data-domain-script="3a6bb61d-8924-4c57-a2b6-bd2306a9c5ef"></Script>

```

Comment:
Can we move the domain specific key to an environment variable please?

```
data-domain-script={process.env.NEXT_PUBLIC_??? || '3a6bb61d-8924-4c57-a2b6-bd2306a9c5ef-test'}
```

and then put the prod value as en env variable in `.github/workflows/deploy.yml`